apps/emqx/etc/emqx.conf

```diff
@@ -1,11 +1,13 @@
 listeners.tcp.default {
   bind = "0.0.0.0:1883"
   max_connections = 1024000
+  limiter.connection = default
```

Comment:
I feel like this line is too hardcore.

Can we write it directly to emqx_schema.erl instead of emqx.conf?

Reply:
@wivwiv  expects the `connection` should have a factory setting, which I don't think is the same as the default value of the field

Reply:
We will change it in the next PR